gallery/src/components/NxSmallTag/NxSmallTagExample.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxSmallTag } from '@sonatype/react-shared-components';
+
+function NxSmallTagExample() {
+  return (
+    <>
+      <div className='nx-small-tag-container'>
```

Comment:
I'm surprised eslint didn't catch this
```suggestion
      <div className="nx-small-tag-container">
```

Reply:
This `div` is no longer needed.

> I'm surprised eslint didn't catch this

According to the doc https://eslint.org/docs/latest/rules/jsx-quotes, should we also add rule `jsx-quotes: ["error", "prefer-double"]` to eslint file?

Reply:
> According to the doc https://eslint.org/docs/latest/rules/jsx-quotes, should we also add rule jsx-quotes: ["error", "prefer-double"] to eslint file?

Yeah seems that way.  Please add a tech debt ticket for it.

Reply:
Created here https://sonatype.atlassian.net/browse/RSC-1723.